add event API details and troubleshooting (#15068)
DataDog/documentation: content/en/logs/guide/sending-events-and-logs-to-datadog-with-amazon-eventbridge-api-destinations.md

    ```
4. Once events and logs are sending, after about five minutes, the data is available in the Datadog [logs console][11] or [events explorer][12], depending on which endpoint you are sending them to.

## Troubleshooting

> To view the response of the API endpoints you can setup AWS SQS for more details regarding the payloads sent to Datadog. 

Reviewer: ```suggestion
To view the response of the API endpoints you can setup Amazon SQS for more details regarding the payloads sent to Datadog. 
```

> 1. Create a new queue in AWS SQS.

Reviewer: ```suggestion
1. Create a new queue in Amazon SQS.
```

> 2. In the target section for your EventBridge rule, expand the additional details section. 

Reviewer: ```suggestion
2. In the **Target** section for your EventBridge rule, expand the **Additional settings** section. 
3. In the **Dead-letter queue** section, choose **Select an Amazon SQS queue in the current AWS account to use as the dead-letter queue**.
```

> 3. Select the AWS SQS created.

Reviewer: ```suggestion
4. Select the AWS SQS created.
5. Update the rule.
```
